### PR TITLE
add thanos ruler

### PR DESCRIPTION
**Pull request by rpahli**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md

### CODE REVIEW

File: charts/prometheus-thanos/out [hunk 1]
```diff
@@ -0,0 +1,377 @@
+---
```
Comment (monotek) [inline charts/prometheus-thanos/out]: superfluous file?

File: charts/prometheus-thanos/templates/ruler-statefulset.yaml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.ruler.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "prometheus-thanos.name" . }}-ruler
+      app.kubernetes.io/instance: {{ .Release.Name }}
+
+  serviceName: dummy-service
```
Comment (monotek) [inline charts/prometheus-thanos/templates/ruler-statefulset.yaml]: servicename not templated?

File: charts/prometheus-thanos/README.md [hunk 3]
```diff
@@ -58,6 +58,13 @@ The following table lists the configurable parameters of the prometheus-thanos c
 | `compact.image.pullPolicy` | Docker image pull policy for store gateway | `IfNotPresent`|
 | `compact.additionalLabels` | Additional labels on compactor pod| `{}`|
 | `compact.additionalAnnotations` | Additional annotations on compactor pod| `{}`|
+| `ruler.replicaCount` |  for store gateway | `1`|
```
Comment (axdotl) [inline charts/prometheus-thanos/README.md]: for `ruler` not `store gateway`
pls. check for all new introduced values.

File: charts/prometheus-thanos/Chart.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.4.0"
 description: A Helm chart for thanos querier and store gateway
```
Comment (monotek) [inline charts/prometheus-thanos/Chart.yaml]: add ruler to describtion

**Pull request merged**